cesiumTestsExtension/bindings/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+include(Macros)
+
+glob_files(SOURCES "${CMAKE_CURRENT_LIST_DIR}/*.cpp")
+
+# cmake-format: off
+setup_python_module(
+    PYTHON_DIR
+        # Use the same Python version as Omniverse (Python 3.10)
+        "${PROJECT_SOURCE_DIR}/extern/nvidia/_build/target-deps/python"
+    TARGET_NAME
+        CesiumOmniverseTestsPythonBindings
+    SOURCES
+        ${SOURCES}
+    LIBRARIES
+        doctest::doctest
```

Comment:
You should be able to remove `doctest::doctest` from here since `doctest::doctest` is a dependency `cesium.omniverse.tests.plugin` and our CMake setup handles the library dependency chain.